### ISSUE TITLE
no slideding if only one image

### ISSUE THREAD

**Issue by micker** [pixedelic/Camera#40]
hello how to stop sliding effect if only one item in slideshow
thanks

**Comment by schipa** [comment]
I'm using following workaround for slideshows with just one slide - automatic generation as well:
```javascript
jQuery(slideshow_selector).camera({
    autoAdvance: false,
    mobileAutoAdvance: false,
    navigation: false,
    playPause: false
});
```
Thus I get that pretty design but this one slide will not advance and no buttons are visible on hover (therefore not accessible). Unfortunately mobile devices are still able to advance/swipe the slide manually when `jquery.mobile.customized.min.js` is included.

**Comment by jeremymouton** [comment]
I ran into this issue as well and decided to fork the repo and fix it.
If anyone is interested: https://github.com/jeremymouton/Camera/commit/c913a6771106c6184f4f2109f255b2e3a009324f

**Comment by gwatsima** [comment]
If i may ask and I'm a novice/still learning. What is the use of only using one image in a slide it sort of beats the purpose of a slide don't you think, why not just use a regular image and make it beautiful or is it the beauty of the slide layout that is alluring. please educate me.

**Comment by jeremymouton** [comment]
@gwatsima, you are right, there is no real value in using a slider for just one image. It would look strange if it rotated by itself every couple seconds.

Though depending on the project, a slider may need to support one or multiple images. If there's only one image, we want to pause or disable the slider so it doesn't slide; and if there are multiple we want the slideshow to start like it would normally.

I needed camera.js to support that scenario, so I setup a fallback to pause the slideshow if only one image is present (as well as hide the pagination dots).

The script is more flexible this way, and behaves properly regardless of how many images are present.

Hope this answers your question.

**Comment by gwatsima** [comment]
Okay i hear you @jeremymouton, but to tell you the truth i find camera.js  to be a little bit tricky for me, i don't know maybe its just me. It works perfectly on some of my sites but refuse completely on others. Maybe i need to understand it more.

**Comment by gwatsima** [comment]
Could you have a solution to this, one of my sites has the logo displaying small because i was constrained by the size of the header div, i don't want to make it bigger but i want the logo to be bigger. The problem I'm encountering is that when i try to adjust the logo it displaces everything below it. Is there a way i can display the logo to overlay the banner div without displacing it. www.leadtimecargo.co.ke

A good example of this is on this website: http://www.chatelaincargo.com
Look at the way the logo overlaps the banner.

**Comment by gwatsima** [comment]
I got a solution thanks

**Comment by mtsweir** [comment]
@schipa has a good work around for this and seems to work fine for me.

You'd still require a few other slider settings though, I found that 'height' and 'loader' where the bare minimum for me to get the slider to actually work. Cheers for posting your solution.
